service/src/main/java/io/camunda/service/entities/DecisionRequirementEntity.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.service.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record DecisionRequirementEntity(
```

Comment:
General question: should we rather use DecisionRequirement**s** with s naming, maybe a question to @ralfpuchert ?

Reply:
even in V1 it seems that `DecisionRequirements` was always taking the plural form and this is not the case of `DecisionDefinition` . 
Even in Zeebe `DecisionRequirements` is used https://github.com/camunda/camunda/blob/572687c78259d56badaac5055164e46f86df52fc/zeebe/gateway-protocol/src/main/proto/gateway.proto#L390
So this is must be for a particular functional reason that I don't know.

Reply:
Thank you all for the clarification 
I am going to refine to `DecisionRequirementsEntity`.